role-make-version-changelog.sh

```diff
@@ -296,6 +297,8 @@ You have three options:
         # When PR are edited using GH web UI, DOS carriage return ^M appears
         # in PR title and body
         sed -i "s/$(printf '\r')\$//" "$rel_notes_file"
+        # $(< file) strips trailing blank lines from a file
+        a=$(<"$rel_notes_file"); printf '%s\n' "$a" > "$rel_notes_file"
```

Comment:
is there somewhere else that adds the single blank line between the new entry and the old entry?